### DISCUSSION TITLE
[Feature Request] - Lists in GitHub Android

### DISCUSSION THREAD

**Discussion by shaunbharat** [community/community#14674] [Mobile] [answered]
I would really like to see lists become a thing in the mobile GitHub app for Android.

It would be really helpful and neat if we could organize our lists of repositories easily, and sort/organize our starred repositories.

Similar to the favourites list on the homepage, maybe all of our lists could be there instead, and maybe the favourites list could be pinned.

**Comment by alcere** [comment, marked as answer]
Hi @ShaunB56, we are working on adding Lists to the Android app, stay tuned for updates!

**Comment by Wlastiiq** [comment]
> I would really like to see lists become a thing in the mobile GitHub app for Android.
> 
> It would be really helpful and neat if we could organize our lists of repositories easily, and sort/organize our starred repositories.
> 
> Similar to the favourites list on the homepage, maybe all of our lists could be there instead, and maybe the favourites list could be pinned.

I wait with you ;)

**Reply by shaunbharat** [reply]
It's here!!!
